### ISSUE TITLE
Authentication server going down disables plugin

### ISSUE THREAD

**Issue by Bunnky** [GC-spigot/AdvancedEnchantments#3240]
### Describe the bug

If the authentication server goes down, AE disables itself. This doesn't make sense for a version that is 100% legal and paid for.

### How to reproduce

Wait for AP's auth server to go down, then try to start a server with AE on it. Or, while the server is up, wait for the auth servers to go down and watch AE disable itself.

### Screenshots / Videos

![image](https://user-images.githubusercontent.com/6545569/231294433-7861f8b9-f13d-4a62-a1f8-06a9666110fa.png)

### "/ae plinfo" link

There is no /ae plinfo, because it's impossible to run this command when the issue arises, simply because the plugin gets disabled entirely

### Server Log
```
[07:32:38] [Craft Scheduler Thread - 264 - AdvancedEnchantments/INFO]: [AdvancedEnchantments] Failed to connect to authentication server - retrying later.
[07:32:38] [Craft Scheduler Thread - 207 - AdvancedPets/INFO]: [AdvancedPets] Failed to connect to authentication server - retrying later.
```

**Comment by GC-spigot** [comment]
Weird, we don't have any outage detected neither do we any other reports / servers from statistics dropping off, also now it runs off cloudflare if there's a problem also doesn't disable in first time connection is lost, make sure everything is good with the server.

**Comment by Bunnky** [comment]
Server is running fine, and has been. We haven't seen this issue in a long time, however the plugin disabling regardless is an issue. Why not implement a token system or something so the plugin doesnt require a server connection to stay enabled? Even if the server is having issues, it's not cool for AP to say "Well your server cant connect so im not gonna work for ya." and just make problems worse.

TLDR; Can we please have a fix for requiring a server to enable **globally**, by means of a config token or something local.
